packages/ui-extensions/src/surfaces/customer-account/components/Section/examples/basic-Section-preact.example.tsx

```diff
@@ -0,0 +1,19 @@
+import {render} from 'preact';
+
+export default function extension() {
+  render(<App />, document.body);
+}
+
+function App() {
+  return (
+    <s-section heading="Rewards">
+      <s-button slot="primary-action" variant="primary">
+        Redeem
+      </s-button>
```

Comment:
Commented on the `shopify-dev` PR too, but maybe we should add a secondary action button here too to match the image?